### ISSUE TITLE
 get_map.py and eval_voc.py

### ISSUE THREAD

**Issue by wasedd** [argusswift/YOLOv4-pytorch#198]
I run my code using voc format dataset training setting as follows:
    "AUGMENT": False,
    "BATCH_SIZE": 2,
    "MULTI_SCALE_TRAIN": True,
    "IOU_THRESHOLD_LOSS": 0.5,
    "YOLO_EPOCHS": 50,
    "Mobilenet_YOLO_EPOCHS": 120,
    "NUMBER_WORKERS": 4,
    "MOMENTUM": 0.9,
    "WEIGHT_DECAY": 0.0005,
    "LR_INIT": 1e-4,
    "LR_END": 1e-6,
    "WARMUP_EPOCHS": 2,  # or None
    "showatt": False
and the val setting as follows:
VAL = {
    "TEST_IMG_SIZE": 416,
    "BATCH_SIZE": 1,
    "NUMBER_WORKERS": 4,
    "CONF_THRESH": 0.005,
    "NMS_THRESH": 0.45,
    "MULTI_SCALE_VAL": False,
    "FLIP_VAL": False,
    "Visual": True,
    "showatt": True
}
and when I run get_map.py ,meet the error.

`Error. File not found: C:\GitHub\test1\utils\input\detection-results\0003.txt
(You can avoid this error message by running extra/intersect-gt-and-dr.py)`

I found there are no detection-results files.How to solve it?

Then I noticed that when I run eval_voc.py.Only evaluation results, no detection results,like this

![屏幕截图 2021-05-24 172308](https://user-images.githubusercontent.com/54884901/119463646-8929b500-bd74-11eb-8ec8-fc68ba73b533.png)
`[2021-05-24 23:24:30,720]-[eval_voc.py line:49]:***********Start Evaluation**************** 
[2021-05-24 23:24:36,078]-[eval_voc.py line:57]:insulator --> mAP : 0.8226071526110587 
[2021-05-24 23:24:36,078]-[eval_voc.py line:57]:knife-2 --> mAP : 0.9758812615955472 
[2021-05-24 23:24:36,078]-[eval_voc.py line:60]:mAP:0.8992442071033029 
[2021-05-24 23:24:36,078]-[eval_voc.py line:61]:inference time: 503.94 ms 
[2021-05-24 23:24:36,078]-[eval_voc.py line:63]:  ===val cost time:5.3581s `

So how to make this part of program work（eval_voc.py）? 
` def detection(self):
        global logger
        if self.__visiual:
            imgs = os.listdir(self.__visiual)
            logger.info("***********Start Detection****************")
            for v in imgs:
                path = os.path.join(self.__visiual, v)
                logger.info("val images : {}".format(path))

                img = cv2.imread(path)
                assert img is not None

                bboxes_prd = self.__evalter.get_bbox(img, v, mode=self.__mode)
                if bboxes_prd.shape[0] != 0:
                    boxes = bboxes_prd[..., :4]
                    class_inds = bboxes_prd[..., 5].astype(np.int32)
                    scores = bboxes_prd[..., 4]

                    visualize_boxes(
                        image=img,
                        boxes=boxes,
                        labels=class_inds,
                        probs=scores,
                        class_labels=self.__classes,
                    )
                    path = os.path.join(
                        cfg.PROJECT_PATH, "detection_result/{}".format(v)
                    )

                    cv2.imwrite(path, img)
                    logger.info("saved images : {}".format(path))`

I can't fix this bug.Please help me.

**Comment by wasedd** [comment]
I have got the pred_result and ground-truth.
![屏幕截图 2021-05-25 163650](https://user-images.githubusercontent.com/54884901/119467068-c93e6700-bd77-11eb-91e2-fb3103727d9d.png)
![屏幕截图 2021-05-25 163415](https://user-images.githubusercontent.com/54884901/119467089-ccd1ee00-bd77-11eb-86a1-2caaf645a0cf.png)

**Comment by lidanyang916** [comment]
I have the same problem，Did you solve it？
